doc/cs_registers.rst

```diff
@@ -46,7 +46,7 @@ Ibex implements all the Control and Status Registers (CSRs) listed in the follow
 +---------+--------------------+--------+-----------------------------------------------+
 |  0x3BF  | ``pmpaddr15``      | WARL   | PMP Address Register                          |
 +---------+--------------------+--------+-----------------------------------------------+
-|  0x7B0  | ``dcsr``           | RW     | Debug Control and Status Register             |
+|  0x7B0  | ``dcsr``           | WARL   | Debug Control and Status Register             |
```

Comment:
Since each field has different behavior, maybe we can remove this here?

Reply:
I would say R is a subset of R/W, and R/W is a subset of WARL. So if, some fields are R only, it is still safe to say that the register is WARL.